guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+
+### More detailed history
+
+You are going to have basic questions like:
+
+ - Who added this piece?
+ - When and why was this added?
+
+And GitHub has built-in tools to do this (one such action is called [git blame](https://help.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file).)
+
+### Easy for many people to collaborate at once
+
+You can work in your own "alternate reality" without interfering with other work, even simultaneously. When ready these alternatives can be merged back together.
+You can also work on changes out of order, although this is more of an advanced topic.
+
+### Web Interfaces
+
+By way of some helpful features, the text files that you are working on can be served as web pages
+directly from where they live. You can read about this feature, called [GitHub Pages](https://pages.github.com/).
+
+To summarize and answer this question, we use GitHub because we want to:
+
+ 1. easily collaborate on common files
+ 2. have advanced reviewing tools and version history
+ 3. keep a record of discussions, and every change made
+ 4. be able to revert back to previous versions, if needed
+ 5. render some of the static files in a web interface
+
+
+## What content do we have on GitHub?
+
+Generally, anything that you'd want to put in a digital safe deposit box is what you'll
+find on GitHub. This obviously includes code, but also includes documentation and
+other web content. Specifically, the [sourcecred organization](https://github.com/sourcecred)
+has the following sets of files. Each set is called a "repository" and you can think of
+it like a (sub)-project under a GitHub organization, or a GitHub username. Each repository
+has it's own link and space on the GitHub website.
+
+ - [sourcecred](https://github.com/sourcecred/sourcecred): the core sourcecred codebase.
+ - [docs](https://github.com/sourcecred/docs): is a growing set of documentation and community health files. In fact, the file that you are reading right now is part of it!
+ - [sourcecred-action](https://github.com/sourcecred/sourcecred-action): a set of workflow that can be used by other GitHub communities.
+
+If you are familiar with filesystems and permissions, you can think of each
+repository as a folder or space that carries it's own permissions, labels,
+and sets of issues, pull requests (requests for changes), milestones, and sometimes
+even guides for contribution. Each is an independent entity under the organization, and thus each
+carries it's own kind of expertise. For example, 
+if you like working with configuration files and workflows, `sourcecred-action`
+might be for you. If you aren't a programmer, you might be most comfortable starting
+with `docs`.
+
+
+### Is my help needed in a software project when I'm not a programmer?
+
+Yes! Any substantial software project is multi-disciplinary. This one included. In fact, calling SourceCred a software project doesn't do justice to how much work besides programming goes into it.
```

Comment:
I super appreciate this insight. It makes me feel super valued and capable of contributing. So thank you :)

Reply:
Definitely! A lot of the stronger (and more fun) communities have a welcoming culture like this, because it really is true that there is a ton of different things to work on. If a community is entirely, say, old school ornery developers, it's going to have a harder time promoting it's message or software to any other kind of person. I've seen it again and again - openness in communication, diversity, and making things fun works really well. As soon as something isn't fun? Why would anyone want to stick around, unless they absolutely had to.

Reply:
Yes. Yes. Andddd yes. I'd be curious to learn which communities are like this to see how successful diverse communities are.

Reply:
Great!